db/enrolled.model.js

```diff
@@ -1,29 +1,11 @@
 
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost:27018/enrolled');
 const Schema = mongoose.Schema;
 
-const db = mongoose.connection;
-db.once('open', _ => {
-  console.log('Database connected');
-});
-
 const enrolledSchema = new Schema({
   enrolled: Number
 });
 
 let Enrolled = mongoose.model('Enrolled', enrolledSchema);
 
-let saveEnrolled = (randomEnrolled) => {
-  const enrolledTable = new Enrolled({
-    randomEnrolled
-  });
-  enrolledTable.save()
-    .then(response => console.log(response))
-    .catch(err => console.log(err));
-};
-
-module.exports = {
-  saveEnrolled,
-  Enrolled
-};
\ No newline at end of file
+module.exports = Enrolled;
```

Comment:
I like that you've put the Model on its own!  I had a bug in my code because I was using two different Models for the seed script and the db methods.  I've since refactored to be just like this.

Reply:
Yup, I did the same in the 1st iteration but realized the mistake. I decided to keep db files only for schema(which I should have from start:) )